src/main/handlers.js

```diff
@@ -352,7 +356,7 @@ function uninstallLinux () {
     'applications',
     'webtorrent-desktop.desktop'
   )
-  fs.removeSync(desktopFilePath)
+  rimraf(desktopFilePath)
```

Comment:
These calls don't need to be `sync` since none of the uninstall functions take callbacks. The caller is just assuming that the handlers will be uninstalled eventually.